.changes/unreleased/NOTES-20231120-152830.yaml

```diff
@@ -0,0 +1,6 @@
+kind: NOTES
+body: 'data-source/http: Includes update to go-retryablehttp fixing preservation of
```

Comment:
Is this a note or bug fix? We typically call out fixes under the appropriate header. 😄

Reply:
Have altered the change log entry to be a bug fix. I'd assumed that since the bug fix was in a dependency rather than in the provider that this would be a note rather than a bug fix in this repo.